extensions/postgres/non_transactional.go

```diff
@@ -72,3 +73,21 @@ func (d dbSession) BatchDeleteWorkerTask(
 	_, err := d.db.ExecContext(ctx, batchDeleteWorkerTaskQuery, filter.ShardId, filter.MinTaskSequenceInclusive, filter.MaxTaskSequenceInclusive)
 	return err
 }
+
+const deleteWorkerTasksQuery = `DELETE
+	FROM xdb_sys_worker_tasks where process_execution_id = $1
+`
+
+func (d dbSession) DeleteWorkerTasks(ctx context.Context, processExecutionId uuid.UUID) error {
```

Comment:
@longquanzheng Which case will the system have a better performance when stopping process execution?

- detele the related worker/timer rows in db so that they will not put into process queue, thus we can save queue storage.
- do nothing so the tasks/timers will go to the process queue but do nothing during execution.